app/charts/column/columns-stacked-state.tsx

```diff
@@ -108,22 +108,15 @@ const useColumnsStackedState = (
     return new Map(values.map((d) => [d.value, d]));
   }, [segmentDimension?.values]);
 
-  const preparedDataGroupedByX = useMemo(
-    () => group(chartData, getX),
-    [chartData, getX]
-  );
-
-  const sumsBySegment = useMemo(
-    () =>
-      Object.fromEntries([
-        ...rollup(
-          scalesData,
-          (v) => sum(v, (x) => getY(x)),
-          (x) => getSegment(x)
-        ),
-      ]),
-    [getSegment, getY, scalesData]
-  );
+  const sumsBySegment = useMemo(() => {
+    return Object.fromEntries([
+      ...rollup(
+        scalesData,
+        (v) => sum(v, (x) => getY(x)),
+        (x) => getSegment(x)
+      ),
```

Comment:
Here we need to spread because fromEntries does not take an iterator ?

Reply:
In fact we don't need to spread, good catch!